### PR TITLE
Release 1.16.0

### PR DESCRIPTION
**Pull request by afallou**
Adds ability to skip a message processed too many times.

### CODE REVIEW

File: microcosm_pubsub/dispatcher.py [hunk 1]
```diff
@@ -81,6 +91,13 @@ def validate_ttl(self):
         if int(self.opaque[TTL_KEY]) <= 0:
             raise TTLExpired()
 
+    def validate_processing_limit(self, message):
+        if self.max_processing_attempts and message.approximate_receive_count > self.max_processing_attempts:
+            raise SkipMessage(
+                "Message exceeded maximum of {max} processing attempts. Skipping",
+                extra=dict(max=self.max_processing_attempts),
+            )
+
```
Comment (wheremiah) [inline microcosm_pubsub/dispatcher.py]: If the uri is not automagically included here, can you add it?

**Comment by wheremiah** [comment]
This sounds like a interesting feature.  Do you have some specific use cases in mind where this is desired, instead of letting the message fail to the DLQ?

**Comment by pelegdvir** [comment]
@wheremiah -
LocalStack doesn't have deadlettering capabilities. To avoid processing the same failing message again and again, we make it possible to just skip it after a certain number of attempts.
(see: https://github.com/globality-corp/microcosm-pubsub/pull/139)

**Comment by wheremiah** [comment]
@pelegdvir 
Hm, but this change applies to the whole platform... I mean I hate DLQs as much as the next guy, maybe more, but this will hide things that should have DLQ'd as SkipMessages.  That seems... risky to me.

**Comment by jessemyers** [comment]
@wheremiah IIUC, the new feature is default disabled (see `@defaults`)

**Pull request merged**